iwyu_ast_util.cc

```diff
@@ -1310,6 +1310,12 @@ bool IsImplicitlyInstantiatedDfn(const FunctionDecl* decl) {
              clang::TSK_ImplicitInstantiation;
 }
 
+const CXXMethodDecl* GetUpmostInHierarchy(const CXXMethodDecl* decl) {
```

Comment:
I don't love this name, but it's not easy to come up with a better one... How about `GetLeastDerived`?

Reply:
`GetFromLeastDerived`, maybe?

Reply:
Sounds good to me!